src/plugins/serial/websocket.js

```diff
@@ -227,6 +238,11 @@ class WebSocketSerial extends BaseSerial {
       console.log('websocket-to-serial: connected');
     } catch (err) {
       console.log('websocket-to-serial: connection failed', err);
+      // Pop an error dialog.  Is there a cleaner way to do this?
+      // Maybe show connection statusi (connected/not connected/connecting) in footer bar?
+      alert('Failed to connect to device.');
```

Comment:
It seems alerts are generally frowned upon but I don't think my javascript skills are up to creating a nice dialog.  I'm happy to change it though if someone wants to give me some pointers